frontend/public/components/dashboard/with-dashboard-resources.tsx

```diff
@@ -234,7 +229,8 @@ export type DashboardItemProps = {
   stopWatchAlerts: StopWatchAlerts;
   urlResults: RequestMap<any>;
   prometheusResults: RequestMap<PrometheusResponse>;
-  alertsResults: RequestMap<PrometheusRulesResponse>;
+  alertsResults: RequestMap<PrometheusRulesResponse>; // TODO: remove once noobaa-storage-plugin, ceph-storage-plugin, and metal3-plugin status cards have been updated to switch over to alertNotifications, see https://github.com/openshift/console/pull/4539
+  notificationAlerts: any;
```

Comment:
There's already a better type in https://github.com/openshift/console/blob/master/frontend/public/components/notification-drawer.tsx#L325 . I think we should move it to `reducers/ui.ts` and reuse it in `notification-drawer.tsx` and `with-dashboard-resources`.

Reply:
Ok, created/moved, pls see last commit for changes -thanks